src/code_f2470_len_27f0.c

```diff
@@ -35,11 +35,35 @@ ApiStatus GetNpcPointer(ScriptInstance* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
+#ifdef NON_MATCHING
+ApiStatus SetNpcPos(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    NpcId npcID = get_variable(script, *args++);
+    f32 x = get_variable(script, *args++);
+    f32 y = get_variable(script, *args++);
+    f32 z = get_variable(script, *args++);
+    Npc* npc = resolve_npc(script, npcID);
+
+    if (npc == NULL) {
+        return ApiStatus_DONE2;
+    }
+
+    npc->pos.x = x;
+    npc->pos.y = y;
+    npc->pos.z = z;
+    npc->colliderPos.x = x;
+    npc->colliderPos.y = y;
+    npc->colliderPos.z = z;
```

Comment:
Can maybe do `npc->pos = { x, y, z }` if that's cleaner (I think so) for these if it matches.

Reply:
I've never done something like this before. Is `{ x, y, z } a struct literal? If so, this is going to generate a struct copy which should be different codegen..but maybe that's why this doesn't match. I'll give it and other things a shot..

Reply:
Yep, it's a struct literal.